chore: open source (#1)
numaproj/numaflow-python: README.md

# Python SDK

> This SDK provides the interface of writing a UDF in Python.

Reviewer: UDSink as well.

# Python SDK

> This SDK provides the interface of writing a UDF in Python.

Reviewer 2: ```suggestion
This SDK provides the interface for writing [UDFs](https://numaproj.github.io/numaflow/user-defined-functions/) and [UDSinks](https://numaproj.github.io/numaflow/sinks/user-defined-sinks/) in Python.
```
